api/specs.proto

```diff
@@ -258,6 +258,12 @@ message ContainerSpec {
 	// task will exit and a new task will be rescheduled elsewhere. A container
 	// is considered unhealthy after `Retries` number of consecutive failures.
 	HealthConfig healthcheck = 16;
+
+	message SecurityConfig {
+	    string userns = 30;
```

Comment:
`userns` and `credential_spec` seem somewhat specific to https://github.com/docker/docker/pull/23389 - wondering if we should have different types of `SecurityConfig`, and this could be one type?

Are there other different security config types do you think go into the `SecurityConfig`?  @diogomonica can security profiles go here, for instance?  (not sure if there are windows specific capabilities that should translate to instead).